backend/src/Designer/Controllers/HomeController.cs

```diff
@@ -131,69 +126,11 @@ public IActionResult Error()
         /// Login
         /// </summary>
         /// <returns>The login page</returns>
+        [Authorize]
```

Comment:
Authentication is configured in a way that [authorize] attribute will redirect to identity providers login page, and after successful login in gitea it will return and redirect to "/".